addons/stock_picking_batch/models/stock_picking_batch.py

```diff
@@ -313,3 +313,12 @@ def _track_subtype(self, init_values):
             return self.env.ref('stock_picking_batch.mt_batch_state')
         return super()._track_subtype(init_values)
 
+    def _is_picking_auto_mergeable(self, picking):
```

Comment:
There is no way to have only one of _is_auto_batchable and _is_picking_auto_mergeable